src/guiguts/maintext.py

```diff
@@ -146,8 +147,23 @@ def __init__(self, parent: tk.Widget, root: tk.Tk, **kwargs: Any) -> None:
         self.frame.columnconfigure(1, weight=1)
         self.frame.rowconfigure(0, weight=1)
 
+        # Set up font
+        family = preferences.get(PrefKey.TEXT_FONT_FAMILY)
+        # If preference has never been set, then choose one of the preferred fonts
+        if not family:
+            families = tk_font.families()
+            for pref_font in "DP Sans Mono", "DejaVu Sans Mono", "Courier":
+                if pref_font in families:
+                    family = pref_font
+                    break
+        self.font = tk_font.Font(
+            family=family,
+            size=preferences.get(PrefKey.TEXT_FONT_SIZE),
+        )
+        # For some reason line spacing on Mac is very tight, so pad a bit here
+        line_spacing = 3 if is_mac() else 0
```

Comment:
If it's okay with @tangledhelix, I'd like to suggest taking this to 4. Since it's specific to Macs, it shouldn't affect anyone else, and I think it looks slightly better. That being said, Dan, if you prefer 3, we can leave it that way.

Reply:
No problem to change it to 4

Reply:
4 looks perfectly fine to me.